etl/steps/grapher/worldbank_wdi/2022-05-26/wdi.py

```diff
@@ -0,0 +1,304 @@
+import re
+import json
+import zipfile
+from typing import Iterable, Optional, List, Union
+from pathlib import Path
+
+import structlog
+import pandas as pd
+
+from owid.catalog import Dataset, Table, Source, VariableMeta
+from owid.catalog.utils import underscore
+from owid.walden import Catalog
+from etl.paths import DATA_DIR
+from etl import grapher_helpers as gh
+from etl.db import get_connection
+
+log = structlog.get_logger()
```

Comment:
Someone other than me is using `structlog`! 🙇

Reply:
haha I think I only started using it after you suggested it on another PR a few months ago :) . Definitely liking it more than `logging`